config/exampleconfiguration.ipynb

```diff
@@ -0,0 +1,101 @@
+{
+ "cells": [
```

Comment:
Did not move this yet. Want /config to be a complete example if anyone wants to grab it for other purposes. Did update it with better comments and an example on how to prevent the config from happening every run.